cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -89,21 +81,68 @@ type CertManagerClient interface {
 
 // certManagerClient implements CertManagerClient .
 type certManagerClient struct {
-	configClient        config.Client
-	proxy               Proxy
-	pollImmediateWaiter PollImmediateWaiter
+	configClient                       config.Client
+	proxy                              Proxy
+	pollImmediateWaiter                PollImmediateWaiter
+	embeddedCertManagerManifestVersion string
+	embeddedCertManagerManifestHash    string
 }
 
 // Ensure certManagerClient implements the CertManagerClient interface.
 var _ CertManagerClient = &certManagerClient{}
 
+func (cm *certManagerClient) getManifestHash() (string, error) {
+	yamlData, err := manifests.Asset(embeddedCertManagerManifestPath)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%x", sha256.Sum256(yamlData)), nil
+}
+
+func (cm *certManagerClient) getManifestVersion() error {
```

Comment:
What about 
```suggestion
func (cm *certManagerClient) setManifestVersion() error {
```
Given that it set a value

Reply:
yes